app/api/books/v1/models.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+books_list = []
+"""
+    List for storing the books added
+"""
+
+class BooksModel():
+    """
+        Class for the books endpoints
+    """
+    def __init__(self):
+        """
+            Constructor method for the class BooksModel
+        """
+        self.db = books_list
+
+    def add_book(self,title,author,category):
+        """
+            Method for serializing the data of a book
+        """
+        status = 'Available'
+        payload = {
+            'status' : status,
+            'title' : title,
+            'author' : author,
+            'category' : category,
+            'id' : len(books_list) + 1,
+            'added_on'  : datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+        } 
+        check_book = self.is_added(title)
```

Comment:
I don't think you need to check if the book title exists since you call the validation in the view already.